packages/synthetic-chain/tsup.config.ts

```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from 'tsup';
+
+export default defineConfig({
+  entry: ['src/cli/cli.ts', 'src/lib/index.ts'],
+  format: ['esm'],
+  dts: true, // Generate declaration file (.d.ts)
+  sourcemap: true,
```

Comment:
Can we generate `d.ts.map` as well? Going to implementation is broken in VS Code without it.

Reply:
Ugh apparently it's a limitation: https://github.com/egoist/tsup/issues/564

Reply:
Actually, it shouldn't be necessary here, I'm confused why go to definition is not working for me